### PR TITLE
Advanced Jim and the Skyscrapers problem solution

### PR DESCRIPTION
**Pull request by michaelmdev**

### CODE REVIEW

File: Data Structures/Advanced/Jim and the Skyscrapers/Solution.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        long k = 0;
+
+        for (int i = 0; i < arr.length; ++i) {
+            int a = arr[i];
+
+            // 1) закрыть свои
+            k+=h[a];
+            ++h[a];
+
+            // 2) прибить меньших
```
Comment (Kvaibhav01) [inline Data Structures/Advanced/Jim and the Skyscrapers/Solution.java]: Please add the English conversion to the comments.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Check the comment.

File: Data Structures/Advanced/Jim and the Skyscrapers/Solution.java (hunk 1 above)
Comment (michaelmdev) [reply Data Structures/Advanced/Jim and the Skyscrapers/Solution.java]: fixed

**Review by Kvaibhav01 — APPROVED** [review]
Good work!

**Pull request merged**